src/libs/kata-sys-util/src/protection.rs

```diff
@@ -14,7 +14,10 @@ use std::path::Path;
 use std::path::PathBuf;
 use thiserror::Error;
 
-#[cfg(any(target_arch = "s390x", target_arch = "powerpc64"))]
+#[cfg(any(
+    target_arch = "s390x",
+    all(target_arch = "powerpc64", target_endian = "little")
+))]
```

Comment:
> powerpc64: Add target_endian = "little"
Based on comments from @Amulyam24 we need to use
the `target_endian = "little"` as well as target_arch = "powerpc64"
to ensure we are working on powerpc64le.

FWIW this was mentioned from the start at https://github.com/kata-containers/kata-containers/issues/10072#issuecomment-2251136045 and also in the first commit of this series.

It is fine to fix `target_endian` everywhere in the same patch.